lib/cloudinary/utils.rb

```diff
@@ -969,7 +969,8 @@ def self.archive_params(options = {})
       :expires_at=>options[:expires_at],
       :transformations => build_eager(options[:transformations]),
       :skip_transformation_name=>Cloudinary::Utils.as_safe_bool(options[:skip_transformation_name]),
-      :allow_missing=>Cloudinary::Utils.as_safe_bool(options[:allow_missing])
+      :allow_missing=>Cloudinary::Utils.as_safe_bool(options[:allow_missing]),
+      :folder_path=>options[:folder_path]
```

Comment:
I don't quite understand where this key is used.

Reply:
@const-cloudinary 
Correct me if I'm wrong, but I believe that according to the reference implementation [this param](https://github.com/cloudinary/cloudinary_npm/blob/7aba2a9f6f8bafe1cb627b70e5f225cdf0b43e25/lib/utils/index.js#L1213) is one of the params used [when creating the params object and signing the request](https://github.com/cloudinary/cloudinary_npm/blob/7aba2a9f6f8bafe1cb627b70e5f225cdf0b43e25/lib/utils/index.js#L1033). I think this change is required, no?

Reply:
@RTLcoil That is a bug, there is no such parameter `folder_path` in the API, it is never passed to the actual API call. You can open an issue on that.